x-pack/plugins/security_solution/server/lib/telemetry/filterlists/prebuilt_rules_alerts.ts

```diff
@@ -352,21 +352,507 @@ export const prebuiltRuleAllowlistFields: AllowlistFields = {
   },
   // Google/GCP
   google_workspace: {
+    actor: {
+      email: true,
```

Comment:
PII check - will this bring back a actual users email address?

Reply:
Potentially, depending if its inbound or outbound.
I can remove to be safe.